content/blog/aws-eks-auto-mode/index.md

```diff
@@ -0,0 +1,2051 @@
+---
+title: "Getting Started with Amazon EKS Auto Mode in Pulumi"
+date: 2024-12-05T22:06:21+01:00
+draft: false
+meta_desc: "Learn how to use Amazon EKS Auto Mode with Pulumi to streamline your Kubernetes cluster management"
+meta_image: meta.png
+authors:
+    - florian-stadler
+tags:
+    - aws
+    - eks
+    - kubernetes
+---
+
+AWS recently announced [Amazon EKS Auto Mode](https://aws.amazon.com/eks/auto-mode/), a significant enhancement to Amazon EKS that streamlines cluster management by automating infrastructure decisions and operations. Today, we are excited to announce that Pulumi fully supports EKS Auto Mode across our AWS provider ecosystem, enabling you to leverage this powerful feature through infrastructure as code.
+
+<!--more-->
+
+## What is EKS Auto Mode?
+
+EKS Auto Mode extends AWS management beyond the Kubernetes control plane to the underlying infrastructure that powers your workloads. It automates critical aspects of cluster operations including:
+
+- Compute provisioning, autoscaling and ongoing optimization
+- Storage configuration and management
+- Load balancer setup and integration
+- Network Policy enforcement
+- OS update and patch management
+
+This automation significantly reduces the operational overhead of running Kubernetes clusters, allowing teams to focus on application development rather than infrastructure management.
+
+## Pulumi Support for EKS Auto Mode
+
+We have added EKS Auto Mode support across our AWS provider ecosystem:
+
+- AWS Native Provider (v6.63.0+)
+- AWS Cloud Control Provider (v1.13.0+)
+- Pulumi EKS Provider (TODO: upcoming release)
```

Comment:
Should we mention AWS Classic here?

Reply:
Uh good catch, the first item is actually classic and I called it native 🤦